sonar-plugin-api/src/main/java/org/sonar/api/measures/MeasuresFilters.java

```diff
@@ -204,6 +204,10 @@ public String filterOnMetricKey() {
     }
   }
 
+  /**
+   * @deprecated since 5.2. Useless by design because of Compute Engine
```

Comment:
would be nice to indicate the alternative (if there is any) or that this concept has just been dropped without replacement. "because of Compute Engine" won't really help a developer